### PR TITLE
Signal handlers

### LINKED ISSUE #303: Install signal handlers in the tlbc-bridge program
We need to explicitly install signal handlers for SIGINT and SIGTERM. When tlbc-bridge is started inside a docker container, both of those signal will be blocked as it's running as pid 1 inside the container. Installing a signal handler will make the program exit immediately when the user runs docker stop, otherwise he will have to wait a few seconds until docker decides to send a SIGKILL.

### PR DESCRIPTION
**Pull request by jannikluhn**
Closes #303

### CODE REVIEW

**Comment by jannikluhn** [comment]
What's wrong with the current approach?

**Comment by schmir-at-bb** [comment]
> What's wrong with the current approach?

It prints a stacktrace (which is not printed without the signal handlers) and the other greenlets don't have a chance to cleanup AFAICT (not that they do anything at the moment).

**Comment by jannikluhn** [comment]
> the other greenlets don't have a chance to cleanup

To make sure I learn something: Don't they already? I assumed they would because we do call `pool.kill` in the `finally` block. If not: Why do we call `pool.kill` in the `finally` block?

Probably to make this more explicit in any case...

File: tools/bridge/bridge/main.py [hunk 1]
```diff
@@ -212,17 +230,15 @@ def main(config_path: str) -> None:
...
-        raise KeyboardInterrupt()
-
     try:
         for coroutine_and_args in coroutines_and_args:
             pool.spawn(*coroutine_and_args)
 
-        gevent.signal(signal.SIGINT, raise_keyboard_interrupt)
-        gevent.signal(signal.SIGTERM, raise_keyboard_interrupt)
+        stop_pool = partial(stop, pool, APPLICATION_CLEANUP_TIMEOUT)
+        gevent.signal(signal.SIGINT, stop_pool)
```
Comment (schmir-at-bb) [inline tools/bridge/bridge/main.py]: You could just replace that with 
```
for signum in (signal.SIGINT, signal.SIGTERM):
    gevent.signal(signum, stop, pool, APPLICATION_CLEANUP_TIMEOUT)
```

But I'm also fine with keeping it the way it is.

**Pull request merged**